Frameworks/UIKit/UIView.mm

```diff
@@ -1405,10 +1395,8 @@ - (void)setFrame:(CGRect)frame {
                      frame.size.height);
     }
 
-    CGRect startFrame = frame;
-
-    if (frame.origin.x == doRound(curFrame.origin.x) && frame.origin.y == doRound(curFrame.origin.y) &&
-        frame.size.width == doRound(curFrame.size.width) && frame.size.height == doRound(curFrame.size.height)) {
+    // Don't bother doing work if there's no work to be done
+    if (memcmp(&frame, &curFrame, sizeof(CGRect)) == 0) {
```

Comment:
This only works if/when the floats in CGRect are packed back-to-back. While that's true now, it might not be universally true. Worth being concerned about?